v0.1/definitions.json

```diff
@@ -56,13 +56,41 @@
       "$id": "#elements/image",
       "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/elements/image.schema.json"
     },
-    "AXES": {
-      "$id": "#options/axes",
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/options/axes.schema.json"
+    "CONTROLLED_TERM": {
+      "$id": "#openminds/controlledTerm",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/CONTROLLED_TERM"
+    },
+    "DATASET_VERSION": {
+      "$id": "#openminds/datasetVersion",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/DATASET_VERSION"
+    },
+    "FILE_INSTANCE": {
+      "$id": "#openminds/fileInstance",
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"
+    },
+    "PROJECT": {
```

Comment:
"PROJECT" cannot exist twice in the dictionary on that level... see general comments of the review.

Reply:
We could rename the second `PROJECT` to `MINDS_PROJECT` or something similar to keep support for the MINDS schemas.

But I agree with you (@lzehl), the best way would probably to just get rid of the MINDS schemas.

Reply:
I would not remove the MINDS schemas just yet. Even though, we will release openMINDS with SANDS, it doesn't mean that all dataset will be migrated to the openMINDS schemas. Wouldn't it be better to give us the chance to add SANDS information to MINDS datasets?

Reply:
To be honest: I do not want to do the work twice and have to deal with workarounds in MINDS v1 because it does not fit a 100% 

I would remove them. It's less work for us and is the cleaner solution.

Reply:
It is of course the cleaner solution, and I'm always for the cleaner solution, but this may limit which dataset can be combined with SANDS. Older dataset, where it might be hard to get the missing information to transfer them into openMINDS, would then be excluded (or postponed for a very long time). 
I would like to discuss this and see how much of a workaround it would actually be to allow linking to MINDS. These are the links to openMINDS:

`protocol|protocolExecution` in:

- `annotation` --> `criteria` (count: 0 - 1)

`controlledTerm` in:

- `annotation` --> `criteriaType` (count: 1)
- `annotation` --> `laterality` (count: 1 - 2)
- `coordinateSpace` --> `anatomicalOrientation` (count: 1 - 1)
- `coordinateSpace` --> `unit` (count: 1 - 1)
- `entityToEntity` --> `confidence` (count: 1 - 1)
- `entityToEntity` --> `relationType` (count: 1 - 1)
- `image` --> `unit` (count: 1 - 1)

`fileInstance` in:

- `atlasTerminology` --> `definedIn` (count: 0 - N)
- `electrodeContact` --> `definedIn` (count: 0 - N)
- `electrodeContact` --> `relatedRecordings` (count: 0 - N)
- `electrodeContact` --> `relatedStimulations` (count: 0 - N)
- `image` --> `definedIn` (count: 1 - 1)

`project` in:

- `brainAtlas` --> `referenceProject` (count: 0 - 1)

`datasetVersion` in:

- `brainAtlasVersion` --> `referenceDataset` (count: 0 - 1)

As far as I can see, the only issue are the `controlledTerm` portions. Annotation criteria aren't required, so it's no problem that we don't have a proper equivalent for `protocol|protocolExecution` in MINDS, and we will transfer our brain atlases to openMINDS anyway. Other dataset, if they need criteria defined, will then need to be transferred to openMINDS first. Otherwise, equivalents for `fileInstance` (--> `file`), `project` (--> `project`) and `datasetVersion` (--> `dataset`) exist in MINDS. 
If we can find a solution for the `controlledTerm` (e.g. mixing MINDS and openMINDS links), I don't see why we shouldn't allow it to give us curators more time to migrate dataset from MINDS to openMINDS but still be able to add SANDS to some of the older datasets.

Reply:
@UlrikeS91 point is that the technical part of this is not discussed yet. I do not know if we actually can link between the two KG versions. Please leave it out for now. We can put it in a controlled way if we really need it. It is yet not clear if the migration to openMINDS really takes "a very long" time.